examples/headsail-bsp/examples/alloc.rs

```diff
@@ -2,17 +2,25 @@
 #![no_main]
 
 extern crate alloc;
-
-use alloc::vec;
+use alloc::vec::Vec;
 use headsail_bsp::{init_alloc, rt::entry, sprintln};
 
+const HEAP_START: usize = 0x1_3000_0000;
```

Comment:
This symbol seems HPC specific, given that it has the HPC-specific external bit. Compile conditionally as appropriate. Alternatively, make sure the whole example is defined HPC specific in Cargo.toml by indicating `required-features = ["hpc"]`.

Reply:
Changes related to this were removed from PR altogether